src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let res = [];
```

Comment:
Try to give variables a more descriptive name
```suggestion
  let previousStates = [];
```